Adding a new curator

ramGranell opened the pull request:
Adding curator required for #2000 
To check this, you need to select the 2022 year (top right) of this page https://deploy-preview-2006--fairsharing.netlify.app/community_curation/our_curators

ramGranell committed:
Adding a new curator

allysonlister commented:
thank you - 2022 looks great! Note that I added a comment earlier today about how we can add her to 2023 as well. thanks!

ramGranell committed:
Update communityCurationCohorts.json

ramGranell commented:
> thank you - 2022 looks great! Note that I added a comment earlier today about how we can add her to 2023 as well. thanks!

Yes, I didn't see that 2023 should be as well. I have just added it.

allysonlister reviewed (approved):
thank you!
